br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -85,6 +87,19 @@ const (
 		);
 	`
 
+	createConflictErrorV2Table = `
+		CREATE TABLE IF NOT EXISTS %s.` + conflictErrorV2TableName + ` (
+			task_id     bigint NOT NULL,
+			create_time datetime(6) NOT NULL DEFAULT now(6),
+			table_name  varchar(261) NOT NULL,
```

Comment:
add index for `task_id, table_name`?

Reply:
Addressed in https://github.com/pingcap/tidb/pull/42647/commits/c5c1efe206a75282e44841476e999d62e87bf671.